config/skylight.yml

```diff
@@ -7,3 +7,4 @@ ignored_endpoints:
   - HeartbeatController#sha
 deploy:
   git_sha: <%= ENV["COMMIT_SHA"] %>
+enable_sidekiq: true
```

Comment:
Let's drop this line:

> If you use Sidekiq with ActiveJob, you can enable either the Sidekiq config or the ActiveJob probe (read on to the next section), or both (it’s not necessary to do both, but it won’t hurt).

I was testing if I could grab the review PR number [here](https://github.com/DFE-Digital/publish-teacher-training/pull/4697), so we could separate the skylight environments per review app. Otherwise all review apps would feed into the same skylight environment.

Reply:
I'm not 100% sure every job uses ActiveJob. In Apply, some jobs are ActiveJob and some are Sidekiq::Workers. I enabled both to make sure both work. I'm pretty sure all our jobs will be Sidekiq but not sure about DfE::Analytics

Reply:
`DfE::Analytics` uses `ActiveJob`: https://github.com/DFE-Digital/dfe-analytics/blob/main/lib/dfe/analytics/analytics_job.rb